Presentation/Presentation/Sources/Chat/View/ChatViewController.swift

```diff
@@ -170,22 +170,26 @@ public final class ChatViewController: UIViewController {
     }
 
     @objc private func keyboardUp(notification: NSNotification) {
-        guard let keyboardFrame: NSValue = notification
-            .userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue else { return }
-        keyboardSize = keyboardFrame.cgRectValue
-        guard let keyboardSize else { return }
-
-        UIView.animate(withDuration: 0.3) { [weak self] in
-            self?.view.frame.size.height -= keyboardSize.height
+        guard
+            let keyboardFrame: NSValue = notification
+                .userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue
+        else { return }
+
+        var keyboardInset: CGFloat = 0
+        if let keyboardHeight {
+            keyboardInset = keyboardFrame.cgRectValue.height - keyboardHeight
+        } else {
+            keyboardInset = keyboardFrame.cgRectValue.height
```

Comment:
```suggestion
        var keyboardInset: CGFloat = keyboardFrame.cgRectValue.height
        if let keyboardHeight {
            keyboardInset -= keyboardHeight
        }
```

이르케 하면 else까지는 없어두 될듯요 !!!